CONTRIBUTING.rst

```diff
@@ -310,39 +361,34 @@ it is currently not possible to retrigger individual tests.
 If opening a PR that changes or adds new APIs, please update or add Python documentation as
 described in `Writing Docs`_ and commit the docs to your PR branch.
 
-Writing Python Tests
-~~~~~~~~~~~~~~~~~~~~
-If your PR includes code that isn't currently covered by our tests (e.g. adding a new flavor, adding
-autolog support to a flavor, etc.), you should write tests that cover your new code. MLflow currently
-uses ``pytest==3.2.1`` for testing. Your tests should be added to the relevant file under ``tests``, or
-if there is no appropriate file, in a new file prefixed with ``test_`` so that ``pytest`` includes that
-file for testing.
+Python Client
++++++++++++++
 
-If your tests require usage of a tracking URI, the
-`pytest fixture <https://docs.pytest.org/en/3.2.1/fixture.html>`_
-`tracking_uri_mock <https://github.com/mlflow/mlflow/blob/master/tests/conftest.py#L74>`_ is automatically set up
-for every tests. It sets up a mock tracking URI that will set itself up before your test runs and tear itself down after.
-If you want to deactivate the mock for your test, mark the test with `@pytest.mark.notrackingurimock` operator.
+For the client, if you are adding new model flavors, follow the instructions below.
 
-Adding New Model Flavor Support
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Python Model Flavors
+--------------------
 
 If you are adding new framework flavor support, you'll need to modify ``pytest`` and Travis configurations so tests for your code can run properly. Generally, the files you'll have to edit are:
 
 1. ``.travis.yml``: exclude your tests in the Windows bash script
```

Comment:
Remove reference to travis here and the line above.

Reply:
Will delete this line if we don't use it. But there's also reference to `NOT_CRAN=true LINTR_COMMENT_BOT=false Rscript ../.travis.R`. What to do about that one?
Any suggestions?

Reply:
done!